lizmap/modules/admin/locales/en_US/admin.UTF-8.properties

```diff
@@ -159,6 +159,7 @@ form.admin_section.message.accessControlAllowOrigin.bad.domain=Content is not a
 menu.lizmap.theme.label=Theme
 configuration.button.modify.theme.label=Modify
 form.admin_theme.h1=Theme
+form.admin_theme.helpcolor=You can select colors with this <a href="https://mdn.github.io/css-examples/tools/color-picker/">colorpicker</a> (e.g. <code>rgb(148 193 31 / 0.5)</code>).
```

Comment:
@laurentj Is it ok to put HTML in strings? If not, how to handle that?

Reply:
It's already done quite few times in LWC.
Transifex manage well HTML in strings, it's a click and click translation process on HTML tags.